Developed in-house at Oracle
Will file a bug upstream asking for this to be a configure option

diff -rupN old/sample/Makefile.am new/sample/Makefile.am
--- old/sample/Makefile.am	2016-02-16 13:53:52.473628366 -0800
+++ new/sample/Makefile.am	2016-02-16 14:14:10.022927698 -0800
@@ -44,7 +44,7 @@
 
 INCLUDES=-I$(top_srcdir)/include
 
-noinst_PROGRAMS = client server
+noinst_PROGRAMS = client server sample-client sample-server
 EXTRA_PROGRAMS = sample-client sample-server
 CLEANFILES=sample-client sample-server ./.libs/*sample-client ./.libs/*sample-server
 
